terraform/modules/aws/waf_v2_acl/configure_aws_wafv2.rb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+
+require 'json'
+require 'logger'
+
+def create_acl!(name, description, rules)
+  out = `aws wafv2 create-web-acl \
+        --scope CLOUDFRONT \
+        --region us-east-1 \
```

Comment:
Is there any benefit to making `region` a variable? I'm thinking in a DR scenario and we want to run this tf to stand up our environment elsewhere?

Reply:
It has to be us-east-1 since that's the "global" region for Cloudfront. Confusing, I know.